zanata-model/src/main/java/org/zanata/hibernate/search/ContainingWorkspaceBridge.java

```diff
@@ -42,8 +42,6 @@ public void set(String name, Object value, Document luceneDocument,
 
         addStringField(IndexFieldLabels.PROJECT_FIELD, project.getSlug(),
                 luceneDocument, luceneOptions);
-        addStringField(IndexFieldLabels.ITERATION_FIELD, iteration.getSlug(),
-                luceneDocument, luceneOptions);
```

Comment:
Why are we getting rid of the indexing by slug? As far as I'm concerned we still need it.